scriptmodules/libretrocores/lr-pcsx-rearmed.sh

```diff
@@ -16,7 +16,11 @@ rp_module_licence="GPL2 https://raw.githubusercontent.com/libretro/pcsx_rearmed/
 rp_module_section="main"
 
 function depends_lr-pcsx-rearmed() {
-    getDepends libpng12-dev libx11-dev
+    if isPlatform "vero4k"; then
+        getDepends libpng-dev
```

Comment:
We handle libpng12-dev -> libpng-dev mapping based on OS version - if you set _os_debian_ver=9 it will do this afair. Once we drop jessie support we can lost all the libpng12-dev for libpng-dev also.

Reply:
Ah yes, I see what you're doing in `helpers.sh`.  It's a bit pedantic here I know, but I was there anyway trying to exclude the unneeded libx11-dev.  In retrospect - based on the x11 issue discussed below - I'll filter that for `x11` platforms then?  And keep the "generic" libpng12-dev for now.  Again it works for me either way, just don't want to break anyone else's builds.